### PR TITLE
Feat/release 2 q2 2020

### PR DESCRIPTION
**Pull request by repjarms**
### Summary

This PR would release version `3.5.0` of the SDK. It contains the following changes:

#### Overall
- Docstring updates across methods

#### AssistantV1
- Additional `MessageInput` model parameters

#### AssistantV2
- Added the `messageStateless` operation and tests

#### VisualRecognitionV4
- Added the `getModel` operation (currently only supports RSCNN for Android) and tests

### CODE REVIEW

File: Source/AssistantV1/Models/MessageInput.swift [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /**
- * (C) Copyright IBM Corp. 2017, 2019.
+ * (C) Copyright IBM Corp. 2017, 2020.
```
Comment (mkistler) [inline Source/AssistantV1/Models/MessageInput.swift:2]: Nice!  I hope the copyright tooling helped you with this.

File: Source/AssistantV2/Models/MessageInputOptions.swift [hunk 2]
```diff
@@ -81,16 +90,18 @@ public struct MessageInputOptions: Codable, Equatable {
      - returns: An initialized `MessageInputOptions`.
      */
     public init(
-        debug: Bool? = nil,
         restart: Bool? = nil,
         alternateIntents: Bool? = nil,
+        spelling: MessageInputOptionsSpelling? = nil,
+        debug: Bool? = nil,
```
Comment (mkistler) [inline Source/AssistantV2/Models/MessageInputOptions.swift:96]: This is a breaking change (changing the position of an existing parameter).  Is that intentional?

Comment (mediumTaj) [reply Source/AssistantV2/Models/MessageInputOptions.swift:96]: @mamoonraja we need to check if this is an issue in all sdks. its fine in .net

Comment (repjarms) [reply Source/AssistantV2/Models/MessageInputOptions.swift]: @mkistler I've updated the parameter position.

**Pull request merged**